src/stores/authenticationStore.ts

```diff
@@ -0,0 +1,55 @@
+import { AccountTypes } from '@bcrs-shared-components/enums'
+import { RootStateIF } from '@/interfaces'
+import { defineStore } from 'pinia'
+
+export const useAuthenticationStore = defineStore('authentication', {
+  getters: {
+    /** Whether the current account is a premium account. */
+    isPremiumAccount (rootState: RootStateIF): boolean {
```

Comment:
Is `rootState` actually the only formal parameter here? In the old code, it was the 3rd.

Reply:
Yes, I believe so. In the old code, the other parameters like `state` and `getters` weren't being used. So, I took them out.

Reply:
I think what you have is OK (but by the way, you can't just remove formal parameters that aren't used -- the 3rd still has to be the 3rd one).